3rdparty/libjpeg-turbo/CMakeLists.txt

```diff
@@ -99,20 +143,81 @@ if(WITH_ARITH_DEC)
   set(JPEG_SOURCES ${JPEG_SOURCES} jdarith.c)
 endif()
 
-# No SIMD
-set(JPEG_SOURCES ${JPEG_SOURCES} jsimd_none.c)
+if(MSVC)
```

Comment:
> MSVC
> MSVC_IDE

Looks like there is some mess in using of these variables.

Reply:
I borrowed these lines from the upstream. Same as `MSVC_IDE` parts.